tests/src/JIT/Directed/RVAInit/nested.Unix.x86.il

```diff
@@ -0,0 +1,254 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+
+
+
+.assembly extern System.Console
+{
+  .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A )
+  .ver 4:0:0:0
+}
+
+.assembly extern mscorlib
+{
+}
+.assembly 'nested'
+{
+}
+.module nested.exe
+.custom instance void [mscorlib]System.Security.UnverifiableCodeAttribute::.ctor() = ( 01 00 00 00 ) 
+
+.file alignment 512
+
+.class public sequential ansi sealed beforefieldinit Internal
+       extends [mscorlib]System.ValueType
+{
+  .field public unsigned int8 _byte
+  .field public int8 _sbyte
+  .field public int16 _short
+  .field public unsigned int16 _ushort
+  .field public int32 _int
+  .field public unsigned int32 _uint
+  .field public int64 _long
+  .field public unsigned int64 _ulong
+  .field public float32 _float
```

Comment:
Instead of duplicating the whole test, I think it would be better to fix the test to not be sensitive to double alignment rules by inserting a dummy padding field here.

Reply:
done